src/views/Events/index.js

```diff
@@ -101,6 +188,7 @@ export default class Events extends Component {
       const events = gordonEvent.getFutureEvents(allEvents); //Filter out past events initially
       this.setState({ allEvents, events, loading: false, filteredEvents: events });
     }
+    this.loadPrevious();
```

Comment:
Why is this added?  A comment explaining the reasoning would help.

Reply:
added comment explaining that this was added to load previously set filters when loading the events either as an authenticated user or as a guest